dev/components/stocks.js

```diff
@@ -1,36 +1,47 @@
 import $ from 'jquery';
-
+import _ from 'lodash';
+import store from 'store2';
+import StockPopUp from './stock-popup.js';
 
 class Stocks {
   constructor(container) {
-
     // REGISTER ELEMENTS
     this.$container = container;
+    this.graph;
+    this.popup;
 
     this.render();
-
     this.$stockListContainer = $('#stocks-list');
-
-    // INITIALIZE FETCH OF STOCKS
-    this.getStocks();
+    this.count = 1;
+
+    // check if local storage exist
+    if (store.get(`stocks${this.count}`)) {
+      this.renderStocks(this.count);
+    }
+    else {
+      this.getStocks(this.count);
+    }
+
+    this.activatePopUp();
+    this.activateScroll();
   }
 
 
   // RENDER HTML
   render() {
     let html =
       `
-        <div class="stocks-canvas">
-          <h3>Add stocks to watchlist</h3>
-          <ul id="stocks-list" class="stocks-list"></ul>
+        <div class="stocks-container">
+          <h3>Stocks</h3>
+          <ol id="stocks-list" class="stocks-list"></ol>
```

Comment:
You may want to consider a loading indicator in here when you first render it in case there are no stocks and you have to fetch data.